app/songflong/__init__.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import uuid
+from pathlib import Path
+from queue import Queue
+from threading import Thread
+from time import time
+
+from .ffmpeg import ffmpeg_merge_video_audio
+
+from .download import download_video_stream, download_audio_stream
+from .links import get_youtube_link, find_all_links
+from .search import get_songs_by_bpm, get_track_bpm
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+logger = logging.getLogger(__name__)
+
+
+def video_links(initial_song):
+    """
+    Finds a list of songs with similar BPM to the given query song.
+
+    :param initial_song: The title of the given song
+    :type initial_song: str
+    :returns: A list of tuples containing the song title, author, and YouTube link
+    :rtype: list
+    """
+    bpm = get_track_bpm(initial_song)
+    songs = get_songs_by_bpm(bpm)
+
+    return get_youtube_link(initial_song, is_audio=False), \
+        [(song, link) for song, link in zip(songs, find_all_links(songs))]
+
+
+def setup_download_dir():
```

Comment:
None of this should be in an init file. Move it to something that we'll easily understand like video_writer.py or something like that